resources/javascript/deno/index.md

```diff
@@ -25,6 +25,8 @@ key_links:
 tutorials:
 - title: 100 second intro (YouTube)
   url: https://youtu.be/F0G9lZ7gecE
+- title: Full Deno Tutorial by Chapters
+  url: https://lyty.dev/deno/deno-tutorial.html
```

Comment:
```suggestion
  url: https://lyty.dev/deno/deno-tutorial.html
  description: The tutorial is broken into sections like "Hello World" and "Package Management".
```

Reply:
"Chapters" are ordered like in a book. These are not numbered and so are more "sections".